tests/phpunit/test-network.php

```diff
@@ -73,4 +73,12 @@ public function test_get_clean_basedomain() {
 	public function test_pantheon_remove_network_setup() {
 		$this->assertNull( Pantheon\NetworkSetup\pantheon_remove_network_setup() );
 	}
+
+	public function test_disable_subdirectory_custom_wp_content_warning() {
+		// Test the default condition.
+		$this->assertNotEmpty( pantheon_get_subdirectory_networks_message() );
+
+		add_filter( 'pantheon.enable_subdirectory_networks_message', '__return_false' );
+		$this->assertEmpty( pantheon_get_subdirectory_networks_message() );
+	}
```

Comment:
```suggestion
	public function test_disable_subdirectory_custom_wp_content_warning() {
		// Test the default condition.
		$this->assertNotEmpty( pantheon_get_subdirectory_networks_message() );

		add_filter( 'pantheon.enable_subdirectory_networks_message', '__return_false' );
		$this->assertEmpty( pantheon_get_subdirectory_networks_message() );
	}

	public function test_disable_subdirectory_custom_wp_content_warning_filtered() {
		add_filter( 'pantheon.enable_subdirectory_networks_message', '__return_false' );
		$this->assertEmpty( pantheon_get_subdirectory_networks_message() );
	}

```
These should be distinct test cases.